beets/dbcore/query.py

```diff
@@ -146,10 +146,9 @@ def __init__(self, field, fast=True):
     def col_clause(self):
         return self.field + " IS NULL", ()
 
-    @classmethod
     def match(self, item):
         try:
-            return item[self.field] is None
+            return item[self.field] is None or not item[self.field]
```

Comment:
I'm a little concerned that `or not item[self.field]` changes the meaning of NoneQuery. For example, it looks like NoneQuery was originally added to find singletons—when `album_id` is null. But if there happens to be an album with id 0, then `not item[self.field]` will be (erroneously) true.

In any case, we do need to make sure that this and the SQL implementation (`col_clause`) do the same thing.